### ISSUE TITLE
[Bug]: 试用towhee打包docker镜像，运行镜像，加载模块失败

### ISSUE THREAD

**Issue by dragonwhl** [towhee-io/towhee#2384]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

pipeline代码：
import towhee

clip_img_embedding_pipe = ( towhee.dummy_input()
        .pipeline_config(format_priority=['tensorrt', 'onnx'])
        .image_decode.cv2_rgb()
        .image_text_embedding.clip_image()
        .as_function()
        )
towhee.build_docker_image(clip_img_embedding_pipe, image_name='clip_image_embedding:v1', cuda='11.4')

打包成docker镜像clip_image_embedding:v1，运行docker镜像，报加载模块错误：
![微信图片_20230204210054](https://user-images.githubusercontent.com/23327686/216769078-d5b775c5-5f34-4a1a-b61a-3c26814b0cd7.png)

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by binbinlv** [comment]
@junjiejiangjjj 
Could you please have a look first? Thanks

**Comment by junjiejiangjjj** [comment]
Will be fixed in towhee 1.0
